packages/nimble-components/src/table/components/row/tests/table-row.spec.ts

```diff
@@ -24,208 +28,254 @@ async function setup(): Promise<Fixture<TableRow<SimpleTableRecord>>> {
-        element.selectable = true;
-        element.selected = false;
-        await connect();
-
-        element.selected = true;
-        await waitForUpdatesAsync();
-
-        expect(element.getAttribute('aria-selected')).toBe('true');
-    });
-
-    it('shows selection checkbox when row is selectable and selection is not hidden', async () => {
-        await connect();
-        element.selectable = true;
-        element.hideSelection = false;
-        await waitForUpdatesAsync();
-
-        expect(element.selectionCheckbox).toBeDefined();
-    });
-
-    it('hides selection checkbox when row is selectable and selection is hidden', async () => {
-        await connect();
-        element.selectable = true;
-        element.hideSelection = true;
-        await waitForUpdatesAsync();
-
-        expect(element.selectionCheckbox).not.toBeDefined();
-    });
-
-    it('hides selection checkbox when row is not selectable', async () => {
-        await connect();
-        element.selectable = false;
-        element.hideSelection = false;
-        await waitForUpdatesAsync();
-
-        expect(element.selectionCheckbox).not.toBeDefined();
-    });
-
-    it('selection checkbox is checked when row is selected', async () => {
-        await connect();
-        element.selectable = true;
-        element.hideSelection = false;
-        element.selected = true;
-        await waitForUpdatesAsync();
-
-        expect(element.selectionCheckbox!.checked).toBeTrue();
-    });
-
-    it('selection checkbox is not checked when row is not selected', async () => {
-        await connect();
-        element.selectable = true;
-        element.hideSelection = false;
-        element.selected = false;
-        await waitForUpdatesAsync();
-
-        expect(element.selectionCheckbox!.checked).toBeFalse();
-    });
-
-    it('selection state can be set before connect()', async () => {
-        element.selectable = true;
-        element.hideSelection = false;
-        element.selected = true;
-        await connect();
-
-        expect(element.selectionCheckbox!.checked).toBeTrue();
-    });
-
-    it('checking selection checkbox fires "row-selection-toggle" event', async () => {
-        element.selectable = true;
-        element.hideSelection = false;
-        element.selected = false;
-        await connect();
-
-        const listener = createEventListener(element, 'row-selection-toggle');
-        element.selectionCheckbox!.click();
-        await listener.promise;
-
-        expect(listener.spy).toHaveBeenCalledTimes(1);
-        const expectedDetails: TableRowSelectionToggleEventDetail = {
-            newState: true,
-            oldState: false
-        };
-        const event = listener.spy.calls.first().args[0] as CustomEvent;
-        expect(event.detail).toEqual(expectedDetails);
-    });
+// prettier-ignore
+async function setupTable(): Promise<Fixture<Table<SimpleTableRecord>>> {
```

Comment:
nit: can you move this down to the `describe('in table',` block it's used in? Makes it easier to see with the tests using it